homeassistant/components/fmi/strings.json

```diff
@@ -0,0 +1,23 @@
+{
+  "config": {
+    "step": {
+      "user": {
+        "description": "Latitude and longitude will be used for weather coordinates",
+        "title": "Connect to the Finnish Meteorological Institute",
+        "data": {
+          "name": "Name",
```

Comment:
Use references in strings, according to https://github.com/home-assistant/core/issues/40578